aladdin.sh

```diff
@@ -263,28 +247,26 @@ function handle_ostypes() {
     esac
 }
 
-function prepare_docker_subcommands() {
+function enter_docker_container() {
+
     # if this is not production or staging, we are mounting kubernetes folder so that
     # config maps and other settings can be customized by developers
+    local -a mounts
     if "$DEV"; then
-        DEV_CMD="-v $(pathnorm "$ALADDIN_DIR"):/root/aladdin"
-        DEV_CMD="$DEV_CMD -v /:/minikube_root" # mount the whole minikube system
-        DEV_CMD="$DEV_CMD --workdir /minikube_root$(pathnorm "$PWD")"
+        mounts=("${mounts[@]}" "-v $(pathnorm "$ALADDIN_DIR"):/root/aladdin")
```

Comment:
I'm not sure this is quite right.  You don't want `"-v $(pathnorm "$ALADDIN_DIR"):/root/aladdin"` to be a single argument, I think --- I think you want "-v" and "$(pathnorm "$ALADDIN_DIR"):/root/aladdin` to be separate arguments.

Reply:
got it, thanks